### PR TITLE
[cbc] Add extractor

### PR DESCRIPTION
**Pull request by jimfunk**

### CODE REVIEW

**Comment by yan12125** [comment]
Since #8497 landed, please move changes in ```youtube_dl/extractors/__init__.py``` to ```youtube_dl/extractors/extractors.py```. Check step 5 of the new [developer instructions](https://github.com/rg3/youtube-dl/blob/master/README.md#adding-support-for-a-new-site) for more information.

File: youtube_dl/extractor/cbc.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        media_id = caffeine_content['entries'][0]['content'][0]['releases'][0]['mediaId'].split('/')[-1]
+
+        caffeine_media = self._download_json(
+            caffeine_media_url + 'q=*&byGuid=' + media_id,
+            video_id
+        )
+
+        episode_info = caffeine_media['entries'][0]
+        from pprint import pprint
+        pprint(episode_info)
```
Comment (yan12125) [inline youtube_dl/extractor/cbc.py]: Please remove debugging lines.

**Comment by yan12125** [comment]
By the way, this PR seems a duplicate of #6342.

**Comment by jimfunk** [comment]
Seems an extractor was already added a little while ago. It has hardcoded IDs, but seems to work at the moment

**Pull request closed without merging**